pkgs/by-name/je/jellyfin-ffmpeg/package.nix

```diff
@@ -8,14 +8,79 @@ let
   version = "7.0.2-5";
 in
 
-(ffmpeg_7-full.override {
+(ffmpeg.override {
   inherit version; # Important! This sets the ABI.
   source = fetchFromGitHub {
     owner = "jellyfin";
     repo = "jellyfin-ffmpeg";
     rev = "v${version}";
     hash = "sha256-cqyXQNx65eLEumOoSCucNpAqShMhiPqzsKc/GjKKQOA=";
   };
+
+  ffmpegVariant = "barebones";
+
+  # https://github.com/NixOS/nixpkgs/pull/354936#issuecomment-2466628369
+  # https://github.com/jellyfin/jellyfin-ffmpeg/blob/jellyfin/debian/rules
+  # --enable-gmp # not supported by our ffmpeg drv
+  withDoc = false;
+  buildFfplay = false;
+  withXcb = false;
+  withSdl2 = false;
+  withXlib = false;
+  withGPL = true;
+  withVersion3 = true;
```

Comment:
Also those are enabled by barebones already. Maybe we should generate this list with a simple grep, sed, awk from the rules file?

Reply:
Those two are not enabled by barebones only in headless and up.